spec/form_spec.cr

```diff
@@ -363,6 +367,16 @@ describe "LuckyRecord::Form" do
         end
       end
     end
+
+    context "with a uuid backed model" do
+      it "doesn't generate a new uuid" do
+        line_item = LineItemBox.create
+        LineItemForm.update(line_item, { "name" => "Another pair of shoes"}) do |form, record|
+          form.saved?.should be_true
+          record.id.should eq line_item.id
+        end
+      end
+    end
```

Comment:
Might be worth adding a test case for manually setting the id with a UUID as mentioned in the comment in form.cr: `UserForm.create!(id: UUID.new("some-hardcoded-uuid"))` should remain unchanged

Reply:
Agreed, I didn't think that manually setting ids was allowed. Test added ✅